wgsl/index.bs

```diff
@@ -7340,23 +7340,20 @@ until the end of the loop body.
 The declaration executes each time it is reached, so each new iteration
 creates a new instance of the [=variable declaration|variable=] or [=value declaration|value=], and re-initializes it.
 
-Note: The loop statement is one of the biggest differences from other shader
```

Comment:
I'm not sure about removing this, the `loop` statement is a big difference from other shader languages. Maybe,

```
Note: The loop statement is a specialized construct, you probably want the `for` or `while` statements. The loop statement is one of the biggest differences from other shader languages.
```

Reply:
added